README.md

```diff
@@ -79,20 +79,22 @@ You only need to get the services stood up and configure the S3 bucket once.
 
 For working with cloud.gov directly, you will need to [install the cloud foundry cli](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). That will allow you to run the `cf` commands in a terminal.
 
-Log on with `cf login -a api.fr.cloud.gov --sso` and go to the link to sign in and get your pass-code.
+First, login to cloud.gov at https://login.fr.cloud.gov/login and then, get a passcode https://login.fr.cloud.gov/passcode.
+
+Log on with `cf login -a api.fr.cloud.gov --sso-passcode <put_passcode_here>` and go to the link to sign in and get your pass-code.
```

Comment:
Seems like this passcode link instruction is repeated above, unless I'm missing something?

Reply:
the instruction above is in a browser the second line is in the command line

Reply:
I see what I was missing - I didn't remove the second log in

Reply:
I see I what I was missing! I need to take out the second login lines since you have to do it before for the passcode variable